### ISSUE TITLE
Update nuget to target dotnetstandard 

### ISSUE THREAD

**Issue by rafamerlin** [ElasticLINQ/ElasticLINQ#120]
Hi, I am not sure how much trouble would be to do that. 

Someone created a different nuget packagethat has done it already, but I am not sure if it's pointing to the latest version of ElasticLinq or if the code is exactly what's here on github.

https://www.nuget.org/packages/ElasticLINQANC/

Cheers

**Comment by damieng** [comment]
No idea what he's done either as it doesn't point to his own repo :(

The portable version should work just fine with dotnetstandard/dotnetcore.

I did go through the whole dotnetstandard migration at some point but I suspect it never got pushed as it doesn't seem to be here any more.

**Comment by rafamerlin** [comment]
I tried to contact the creator of the ElasticLINQANC package to see if he can create a PR here to do that.

I was a bit concerned as I'll run my app in dotnetcore 2.0 in linux, and I was getting this warning while in dev:
`Warning	NU1701	Package 'ElasticLinq 1.4.3' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.0'. This package may not be fully compatible with your project.	`

But I managed to deploy it in a linux container in AWS and it's working fine.
